README.md

```diff
@@ -354,8 +354,12 @@ Subscribe to an event published by the widget.
 - `context` - Optional context to bind the callback to
 
 ```javascript
-signIn.on('pageRendered', function (data) {
-  console.log(data);
+signIn.on('navigated', function (context) {
+  console.log(context);
+  // {
+  //   view: '/',
+  //   controller: 'primary-auth'
+  // }
```

Comment:
- add `transaction` field
- I wonder if it's worth to use a actual event as example here cause it's sort of duplicated with "events" section. Something like

```
 - `event` - [Event](#events) to subscribe to
 - `callback` - Function to call when the event is triggered. That function takes one parameter optional `context`. More details see [Event](#events).

signIn.on('EVENT_NAME', function (context) { ... });

```

Reply:
Great idea. I'm going to shorten the content in this section, and point our readers to the `#Events` section.